auslib/blobs/apprelease.py

```diff
@@ -955,6 +971,15 @@ def __init__(self, **kwargs):
         Blob.__init__(self, **kwargs)
         if 'schema_version' not in self.keys():
             self['schema_version'] = 9
+        self['has_wnp'] = self.has_wnp
+
+    @property
+    def has_wnp(self):
```

Comment:
As a suggestion to not reimplement the entire logic, you can turn `has_wnp` to a simple function with a param "wnp_field":
`has_wnp(self, wnp_field)`